docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -428,25 +429,26 @@ capacityAlerts:
         TooMuchMemoryRequested:
           enabled: true
           expr:
-            raw: sum(kube_pod_resource_request{resource="memory"}) > 9000*1024*1024*1024 <5>
+            raw: sum(kube_pod_resource_request{resource="memory"}) > 9000*1024*1024*1024 <6>
     CpuCapacity:
       rules:
         ClusterCpuUsageHigh:
-          enabled: false <6>
+          enabled: false <7>
         ExpectClusterCpuUsageHigh:
-          enabled: false <6>
+          enabled: false <7>
     UnusedCapacity:
       rules:
         ClusterHasUnusedNodes:
-          enabled: false <7>
+          enabled: false <8>
 ---
 <1> Enables capacity alerts
-<2> Changes the alert message for the pod capacity alert
-<3> Only alerts for pod capacity if it fires for 3 hours
-<4> Change the pod count prediction to look at the last two days and predict the value in five days
-<5> Completely overrides the default alert rule and alerts if the total memory request is over 9000 GB
-<6> Disables both CPU capacity alert rules
-<7> Disables alert if the cluster has unused nodes.
+<2> List of node labels (as they show up in the `kube_node_labels` metric) by which alerts are segregated
```

Comment:
Segregated sounds weird to me (and has relatively negative connotations in general)

```suggestion
<2> List of node labels (as they show up in the `kube_node_labels` metric) by which alerts are grouped
```